GitUI/Editor/FileViewer.cs

```diff
@@ -374,6 +374,16 @@ public ArgumentString GetExtraDiffArguments(bool isRangeDiff = false)
             };
         }
 
+        public ArgumentString GetExtraGrepArguments()
+        {
+            return new ArgumentBuilder
+            {
+                "-h",
+                { ShowEntireFile, "--context=9000", $"--context={NumberOfContextLines}" },
```

Comment:
special magic number, let's use a more insane value
```
int numberOfContextLines = ShowEntireFile ? 100_000 : NumberOfContextLines;
```

Reply:
OK, this is not changed from before